packages/nextjs/.eslintrc.json

```diff
@@ -5,6 +5,7 @@
     "@typescript-eslint/no-unused-vars": ["error"],
     "@typescript-eslint/no-explicit-any": ["off"],
     "@typescript-eslint/ban-ts-comment": ["off"],
+    "@next/next/no-img-element": ["off"],
```

Comment:
Was getting a linting error because of this rule

Reply:
Yeah, we have that so people use NextJS <Image /> which comes with [some optimizations](https://nextjs.org/docs/app/building-your-application/optimizing/images)

```
<Image alt="github icon" width="24" height="24" src="/icon-github.svg" />
```

Not a big deal, especially for these tiny images